tables/ADDRINDX.sql

```diff
@@ -1,68 +1,68 @@
-  `adrpre` varchar(10), 
-  `adrno` decimal(10,0), 
-  `adrgrid` varchar(12), 
-  `adradd` varchar(10), 
-  `adrdir` varchar(2), 
-  `adrstr` varchar(50), 
-  `adrsuf` varchar(8), 
-  `adrsuf2` varchar(8), 
-  `cityname` varchar(50), 
-  `statecode` varchar(2), 
-  `unitdesc` varchar(20), 
-  `unitno` varchar(20), 
-  `zip1` varchar(5), 
-  `zip2` varchar(4), 
-  `loc2` varchar(40), 
-  `defaddr` varchar(1), 
-  `deactivat` string, 
-  `iasw_id` decimal(10,0), 
-  `trans_id` decimal(10,0), 
-  `upd_status` varchar(1), 
-  `gislink` varchar(20), 
-  `bldgno` varchar(10), 
-  `childparid` varchar(30), 
-  `user1` varchar(40), 
-  `user2` varchar(40), 
-  `user3` varchar(40), 
-  `user4` varchar(40), 
-  `user5` varchar(40), 
-  `user6` varchar(40), 
-  `user7` varchar(40), 
-  `user8` varchar(40), 
-  `user9` varchar(40), 
-  `user10` varchar(40), 
-  `user11` varchar(40), 
-  `user12` varchar(40), 
-  `user13` varchar(40), 
-  `user14` varchar(40), 
-  `user15` varchar(40), 
-  `adrid` decimal(10,0), 
-  `adrparchild` varchar(1), 
-  `adrstatus` varchar(2), 
-  `adrpremod` varchar(20), 
-  `adrpretype` varchar(20), 
-  `adrpostmod` varchar(20), 
-  `floorno` varchar(20), 
-  `coopid` varchar(30), 
-  `country` varchar(30), 
-  `postalcode` varchar(10), 
-  `addrsrc` varchar(10), 
-  `addrvalid` varchar(1), 
-  `xcoord` decimal(10,0), 
-  `ycoord` decimal(10,0), 
-  `strcd` varchar(10), 
-  `strreloc` varchar(150), 
+  `jur` varchar(6),
+  `parid` varchar(30),
+  `card` decimal(4,0),
+  `lline` decimal(4,0),
+  `tble` varchar(30),
+  `tabseq` decimal(3,0),
+  `seq` decimal(3,0),
+  `cur` varchar(1),
+  `who` varchar(50),
+  `wen` string,
+  `adrpre` varchar(10),
+  `adrno` decimal(10,0),
+  `adrgrid` varchar(12),
+  `adradd` varchar(10),
+  `adrdir` varchar(2),
+  `adrstr` varchar(50),
+  `adrsuf` varchar(8),
+  `adrsuf2` varchar(8),
+  `cityname` varchar(50),
+  `statecode` varchar(2),
+  `unitdesc` varchar(20),
+  `unitno` varchar(20),
+  `zip1` varchar(5),
+  `zip2` varchar(4),
+  `loc2` varchar(40),
+  `defaddr` varchar(1),
+  `deactivat` string,
+  `iasw_id` decimal(10,0),
+  `trans_id` decimal(10,0),
+  `upd_status` varchar(1),
+  `gislink` varchar(20),
+  `bldgno` varchar(10),
+  `childparid` varchar(30),
+  `user1` varchar(40),
+  `user2` varchar(40),
+  `user3` varchar(40),
+  `user4` varchar(40),
+  `user5` varchar(40),
+  `user6` varchar(40),
+  `user7` varchar(40),
+  `user8` varchar(40),
+  `user9` varchar(40),
+  `user10` varchar(40),
+  `user11` varchar(40),
+  `user12` varchar(40),
+  `user13` varchar(40),
+  `user14` varchar(40),
+  `user15` varchar(40),
+  `adrid` decimal(10,0),
+  `adrparchild` varchar(1),
+  `adrstatus` varchar(2),
+  `adrpremod` varchar(20),
+  `adrpretype` varchar(20),
+  `adrpostmod` varchar(20),
+  `floorno` varchar(20),
+  `coopid` varchar(30),
+  `country` varchar(30),
+  `postalcode` varchar(10),
+  `addrsrc` varchar(10),
+  `addrvalid` varchar(1),
+  `xcoord` decimal(15,8),
+  `ycoord` decimal(15,8),
```

Comment:
`coord` fields are just `NUMBER` in Oracle, which will apparently just store the number as originally entered. When pulling via sqoop, `NUMBER` becomes the default Hive `decimal(10,0)`, so we adjust it manually here to account for the precision of these coords.

We don't need to do this for most fields since they already have a precision and scale specified.